cmake/TaichiCAPITests.cmake

```diff
@@ -23,6 +23,7 @@ target_include_directories(${C_API_TESTS_NAME}
     ${PROJECT_SOURCE_DIR}
     ${PROJECT_SOURCE_DIR}/c_api/include
     ${PROJECT_SOURCE_DIR}/c_api/src
+    ${PROJECT_SOURCE_DIR}/external/Vulkan-Headers/include
```

Comment:
Wait `taichi/taichi.h` will be released to the users. Does that mean users also have to add this include `${PROJECT_SOURCE_DIR}/external/Vulkan-Headers/include` in their Demo in order to use C-API with Vulkan?

Reply:
I was thinking C-API should be self-contained, is it because we re-used some Vulkan data types in C-API interface?

Reply:
The user can opt out the use of `taichi/taichi_vulkan.h` if they don't need vulkan interop; but in case they want it, they will have Vulkan SDK installed and the headers are guaranteed forward compatible.